datanode/gateway/graphql/marshallers/marshallers.go

```diff
@@ -284,7 +284,18 @@ func MarshalOrderType(s vega.Order_Type) graphql.Marshaler {
 }
 
 func UnmarshalOrderType(v interface{}) (vega.Order_Type, error) {
-	return vega.Order_TYPE_UNSPECIFIED, ErrUnimplemented
+	s, ok := v.(string)
+	if !ok {
+		return vega.Order_Type(vega.Order_TIME_IN_FORCE_UNSPECIFIED), fmt.Errorf("expected order type to be a string")
```

Comment:
shouldn't that be vega.Order_TYPE_UNSPECIFIED?

Reply:
addressed.